timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -137,7 +137,34 @@ unblocks other transactions, and materializes the aggregates. The first
 transaction is very quick, and most of the work happens during the second
 transaction, to ensure that the work does not interfere with other operations.
 
+## Using continuous aggregates in a multi-node environment
+If you are using Timescale in a multi-node environment, there are some
+additional considerations for continuous aggregates.
+
+When you create a continuous aggregate within a multi-node environment, the
+continuous aggregate should be created on the access node. While it is possible
+to create a continuous aggregate on data nodes, it will interfere with the
+continuous aggregates on the access node and will cause problems with the
+continuous aggregates on the access node.
```

Comment:
Some repetition, sorry about that. Not sure if we should avoid the repetition earlier as well.
```suggestion
continuous aggregates on the access node and will cause problems on the access node.
```

Reply:
Ugh, just noticed the 'wills' as well.